pigar/parser.py

```diff
@@ -14,6 +14,11 @@
 from .helpers import parse_git_config, trim_suffix
 
 import nbformat
+if sys.version_info.major == 3:
+    PYTHON_VERSION_3 = True
+    import IPython
+else:
+    PYTHON_VERSION_3 = False
```

Comment:
What happens if Python 4.x gets released?

I think this is better:
```python
try:
    import IPython
    _transformer_manager = IPython.core.inputtransformer2.TransformerManager()
except ImportError:
    _transformer_manager = None
```

Reply:
Should I remove the comment? I don't have any idea for the comment nor the `Exeption` class.

```
    except Exception:  # ???
```